### PR TITLE
[Assignment 3] 100% coverage

### PR DESCRIPTION
**Pull request by adrian-tiberius**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -10,19 +10,20 @@ var Version = require('./version');
...
 
-            Hoek.assert(!err, err);
-            console.log('Server started at: ' + server.info.uri);
+            return callback(err, server);
         });
     });
 };
 
-internals.init();
+exports = module.exports = internals.init;
```
Comment (chyld) [inline lib/index.js]: a much simpler version would be exports.init = function(port, callback){}. and that's it. you're done.

File: start.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+var Server = require('./lib/index');
```
Comment (chyld) [inline start.js]: or ```var Server = require('./lib')``` index.js is the default file.

Comment (AdriVanHoudt) [inline start.js]: being more specific can't hurt though

File: lib/index.js [hunk 3]
```diff
@@ -10,19 +10,20 @@ var Version = require('./version');
 var internals = {};
```
Comment (hussainanjar) [inline lib/index.js]: You are still requiring Hoek though its not used any more .. made the same mistake :smile:

File: test/index.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        Server(internals.port, function(err, server) {
+
+            expect(!err).to.equal(true);
+            expect(server.info.port).to.equal(internals.port);
+
+            server.stop(done);
+        });
+    });
+
+    it('test plugin error', function (done) {
```
Comment (hussainanjar) [inline test/index.js]: Monkey patching code needs { parallel: false } in the `it('test plugin error', { parallel: false }, function (done) {`
See @hueniverse comments made in @chyld PR #80 and the main issue.

**Comment by hussainanjar** [comment]
You need to add new line to almost all your files, why new line ? This may help http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline

**Pull request closed without merging**